core/src/test/scala/com/salesforce/op/stages/impl/feature/OPMapVectorizerTest.scala

```diff
@@ -166,7 +166,7 @@ class OPMapVectorizerTest extends FlatSpec with TestSparkContext with AttributeA
     testFeatureToMap[Email, EmailMap, String](emailData, emailData2, emailData3)
   }
 
-  "ID features" should "be vectorized the same whether they're in maps or not" in {
+  ignore should "vectorize ID features the same whether they're in maps or not" in {
```

Comment:
please put a TODO here to reenable this (even if you are planning to do so in the next PR)